src/checkpoint.rs

```diff
@@ -64,6 +65,25 @@ impl<'db> Checkpoint<'db> {
         }
         Ok(())
     }
+
+    /// Exports all live SST files of a specified Column Family onto export_path,
+    /// returning SST files information in metadata.
+    pub fn export_column_family<P: AsRef<Path>>(
+        &self,
+        column_family: &impl AsColumnFamilyRef,
+        export_path: P,
+    ) -> Result<ExportImportFilesMetaData, Error> {
+        let cpath = to_cpath(export_path)?;
+        let column_family_handle = column_family.inner();
+        let metadata = unsafe {
+            ffi_try!(ffi::rocksdb_checkpoint_export_column_family(
+                self.inner,
+                column_family_handle,
+                cpath.as_ptr(),
+            ))
```

Comment:
Will this trigger flushing the memtables? Is this something that is controllable?

Reply:
According to https://github.com/facebook/rocksdb/blob/295326b6ee3c895b5f093d1fd44e09c23050ea72/include/rocksdb/utilities/checkpoint.h#L52-L60, it will always flush.

Reply:
Indeed it does; there doesn't appear to be a way in the underlying API to decouple flushing or checkpoint creation from CF export.